documentation/correctness/issues.adoc

```diff
@@ -0,0 +1,96 @@
+= Issues
+
+It makes sense to phrase the answer as a collection of _issues_, instead of giving a hard "`yes/no`" answer.
+An **issue** is a specific point in a serialization chunk where a specific kind of incorrectness arises.
+
+Many (kinds of) issues allow a meaningful (full or partial) _recovery_ — i.e.: they can be recovered from.
+Recovery is often obvious enough that it can be performed programmatically — i.e.: automatically.
```

Comment:
I find resulting text like `[...] programmatically — i.e.: automatically. E.g., an illegal [...]` _very_ hard to parse.